hw/top_earlgrey/data/ip/chip_aon_timer_testplan.hjson

```diff
@@ -0,0 +1,125 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_aon_timer
+  testpoints: [
+    {
+      name: chip_sw_aon_timer_wakeup_irq
+      desc: '''Verify the AON timer wake up interrupt in normal operating state.
+
+            - Program the PLIC to let the AON timer wake up interrupt the CPU.
+            - Program the AON timer to generate the wake up timeout interrupt after some time.
+            - Issue a WFI to wait for the interrupt to trigger.
+            - Service the interrupt when it triggers; verify that it came from AON timer.
+            - Verify that the interrupt triggered only after the timeout elapsed.
+            '''
+      stage: V2
+      si_stage: SV2
+      features: ["AON_TIMER.INTERRUPT.WAKEUP"]
```

Comment:
Add the list of lc_stages you intend to test this in. Obviously this requires the CPU to be enabled.